packages/flutter/lib/src/cupertino/localizations.dart

```diff
@@ -343,7 +343,7 @@ class DefaultCupertinoLocalizations implements CupertinoLocalizations {
   const DefaultCupertinoLocalizations();
 
   /// Short version of days of week.
-  static const List<String> shortWeekdays = <String>[
+  static const List<String> _shortWeekdays = <String>[
```

Comment:
Technically, this is a breaking change, but I don't know if this is worth deprecating? I don't know why somebody would be using a static list of english only day names from a localization class, but if they are maybe we let it break their app, and they can just make their own static list.